dashboard/app/models/pd/workshop.rb

```diff
@@ -183,6 +183,10 @@ def self.in_state(state)
     end
   end
 
+  def course_name
+    course == Pd::Workshop::COURSE_ADMIN ? 'Administrator' : course
+  end
+
```

Comment:
I agree this is hacky. I also think it's valuable separate the keys we use in the database from the exact text we show on the screen, in fact in most places we are forced to do this so we can translate the strings. How about something like this?
```
COURSE_NAMES = {
  Pd::Workshop::COURSE_ADMIN: 'Administrator',
  Pd::Workshop::COURSE_COUNSELOR: 'Counselor',
  ...
}
def course_name
  COURSE_NAMES[course]
end
```